interface-definitions/service_dns_forwarding.xml.in

```diff
@@ -793,6 +793,155 @@
                   </leafNode>
                 </children>
               </node>
+              <tagNode name="zone-to-cache">
+                <properties>
+                  <help>Load a zone into the recursor cache</help>
+                  <valueHelp>
+                    <format>txt</format>
+                    <description>The name of the zone to load</description>
```

Comment:
I don't think we should use such wordy help strings. Just "Domain name" would suffice.

Reply:
done